web-basics-browser-security-model/activities/king-kong/README.md

```diff
@@ -0,0 +1,27 @@
+# Name: King-Kong
+
+## Description
+
+Get the flag from [b]http://141.85.224.118:8086/king-kong/[/b].
+
+Score: 25
+
+## Vulnerability
+
+The flag is displayed only if you send a request with User-Agent change to King-Kong.
+
+## Exploit
+
+Script in `./sol/solution`
```

Comment:
The `sol/` folder does not appear in GitHub changed files but it appears locally, weird 🤔